src/relax/op/ccl/ccl.h

```diff
@@ -35,6 +35,9 @@ namespace relax {
 /*! \brief AllReduce. */
 Expr allreduce(Expr data, String op_type);
 
+/*! \brief Broadcast data from worker-0 to all other workers. */
+Expr broadcast_from_zero(Expr data);
```

Comment:
Let’s think of a consistent naming for this primitive across Relax and Disco/NCCL. broadcast_from_worker0 or broadcast_from_zero?

Reply:
I prefer broadcast_from_worker0, which is more straightforward. But I'm not sure if it's ok to have numbers in function names.

Reply:
It’s fine to have numbers

Reply:
agree about broadcast_from_worker0

Reply:
cool. @LeshengJin please update the PR to reflect the name consistency and let's get it merged today!

Reply:
Updated!